test/client-checkout-integration-test.js

```diff
@@ -175,6 +238,25 @@ suite('client-checkout-integration-test', () => {
     });
   });
 
+  test('it resolves with user errors on Client.checkout#updateLineItems when variant is invalid', () => {
+    fetchMock.postOnce(apiUrl, checkoutLineItemsUpdateWithUserErrorsFixture);
```

Comment:
For consistency with other tests, could we move this line below the `const` declarations? Same for the test below